Feature/jubilacion

Br3nda opened the pull request:
* Tax and benefit system evolution.
Added retirement age (`edad_de_jubilacion` and pension eligibility `eligible_for_pension`

closes https://github.com/ServiceInnovationLab/Piccolo/issues/9

Br3nda committed:
Removing unused stuff

Br3nda committed:
Add the age of retirement

Br3nda committed:
Jubilacion

Br3nda committed:
Tests for jubilacion

Br3nda committed:
Remove the deploy step

Br3nda committed:
No need to import genders

Br3nda committed:
Lint fix up

Br3nda committed:
Lint fix up

Br3nda committed:
Comment out version check

dlouise64 commented on openfisca_uruguay/parameters/general/age_of_retirement.yaml:
```diff
@@ -1,10 +1,8 @@
+description: Edad de jubilacion (in years)
 metadata:
-  reference: https://fr.wikipedia.org/wiki/Retraite_(économie)
+  reference: https://legislativo.parlamento.gub.uy/temporales/leytemp5815466.htm
```
@Br3nda 404 is returned on this url

dlouise64 commented on openfisca_uruguay/variables/benefits.py:
```diff
@@ -1,69 +1,40 @@
-        age_condition = person('age', period) >= parameters(period).general.age_of_majority
-        return age_condition * parameters(period).benefits.basic_income  # This '*' is a vectorial 'if'. See https://openfisca.org/doc/coding-the-legislation/30_case_disjunction.html#simple-multiplication
+    label = "Eligiblity for pension"
+    reference = "https://legislativo.parlamento.gub.uy/temporales/leytemp5815466.htm"
```
@Br3nda 404 is returned on this url

dlouise64 commented on openfisca_uruguay/variables/children.py:
```diff
@@ -0,0 +1,11 @@
+    value_type = int
+    entity = Person
+    definition_period = MONTH
+    label = "Number of years worked"
```
@Br3nda Label should say Number of children

Br3nda committed:
Fixed incorrect label "Number of children"

Br3nda committed:
Fixed incorrect label "Number of children"

Br3nda committed:
Fix link to legislation

this was a 404, but now we'll link to policy instead until we can
find a public legislation website
